packages/block-editor/src/components/global-styles/typography-panel.js

```diff
@@ -130,13 +130,30 @@ function useHasTextDecorationControl( name, element ) {
 	return supports.includes( 'textDecoration' );
 }
 
+function TypographyToolsPanel( { ...props } ) {
+	return <ToolsPanel label={ __( 'Typography' ) } { ...props } />;
+}
+
+const DEFAULT_CONTROLS = {
+	fontFamily: true,
+	fontSize: true,
+	fontAppearance: true,
+	lineHeight: true,
+	letterSpacing: true,
+	textTransform: true,
+	textDecoration: true,
+};
```

Comment:
It is interesting to note here that the "default controls" are sourced from the block.json for the block inspector but they are actually "static" for the global styles. In other words, it's an inconsistency between say "button" panel in global styles and "button" panel in the block inspector.

I've considered removing the "defaultControls" prop and just "always" sourcing from the block.json but I thought maybe we should not change any behavior in the current PR and just leave this unification for its own PR.